public/data/mohit1607.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Mohit Ashliya",
+  "type": "personal",
+  "bio": "Yo this is Superset for OpenSource Contribution.",
+  "avatar": "https://github.com/SuperSet1607.png",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/mohit1607",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ashliya_mohit",
+      "icon": "twitter"
+    }
+  ],
+  "milestones":[
+    {
+      "title":"Started Open Source Contribution",
+      "data":"December 2021",
+      "icon":"dollor",
+      "color":"grey",
+      "description":"From a day before 2023, I started my open source journey."
```

I think you meant =>
```suggestion
      "description": "From a day before 2022, I started my open source journey."
```